app/notifications/validators.py

```diff
@@ -144,10 +144,19 @@ def check_if_service_can_send_to_number(service, number):
         return international_phone_info
 
 
-def check_content_char_count(template_with_content):
+def check_content_character_count(template_with_content):
```

Comment:
Optional: The function name is better however this function checks character counts for sms and checks bytes for email which means the function name is a little misleading.

I don't think we can make the link back from bytes to characters though unfortunately (which would give a nicer message for the user). Maybe `check_message_is_not_too_long` or `check_message_length` would be better?

It might be worth tagging Karl for a quick review here too please.

Reply:
I like that idea, I did send a Slack message to @karlchillmaid about this PR when I first put it up ☺️

Reply:
Ah, great minds think alike then Pea 👍